### ISSUE TITLE
Faulty Fault Button

### ISSUE THREAD

**Issue by davidetome** [robotology/icub-tech-support#1124]
### Robot name 🤖

iCubGenova09

### Request/Failure description

Fault button not working

### Detailed context

I noticed that the robot behaves as the fault button is always released even if physically detached.

### Additional context

cc @pattacini @S-Dafarra @Fabrizio69

**Comment by pattacini** [comment]
This is a quite blocking issue, unfortunately.

**Comment by davidetome** [comment]
cc @isorrentino

**Comment by Fabrizio69** [comment]
Next Friday @emilianob80 can support @violadelbono for this issue

**Comment by violadelbono** [comment]
The fault issue was solved and we found the following problems:

- The fault cable on the BAT board was connected to the wrong connector
- The left leg caused the fault to be kept at ground. Probably due to a cover screw that we noticed pinching the cable

The leg cabling was fixed and the connector on the BAT was switched in the correct place. The robot was tested and now the fault button is fully working.

cc @MrAndrea
